backend/vaa-strapi/src/extensions/users-permissions/strapi-server.js

```diff
@@ -59,16 +66,11 @@ module.exports = async (plugin) => {
       });
     }
 
-    // Setup email template (left for the future, the default template also does not make the URL clickable)
+    // Setup email template (the default template also does not make the URL clickable)
     const email = await pluginStore.get({key: 'email'});
-    // All options can be found here:
-    // https://github.com/strapi/strapi/blob/2a2faea1d49c0d84077f66a57b3b73021a4c3ba7/packages/plugins/users-permissions/server/bootstrap/index.js#L41-L79
-    email.reset_password.options.message = `<p>We heard that you lost your password. Sorry about that!</p>
-
-<p>But don’t worry! You can use the following link to reset your password:</p>
-<a href="<%= URL %>?code=<%= TOKEN %>"><%= URL %>?code=<%= TOKEN %></a>
-
-<p>Thanks.</p>`;
+    email.reset_password.options.message = fs
+      .readFileSync('config/email-templates/reset-password.html')
+      .toString();
```

Comment:
So, is this change intended to solve the clickable URL problem?

Reply:
One of the goals, yes. Additionally, this allows easy modification of the template to the application's administrator as requested during a sprint review (compared to being hardcoded in the code).